test/index.js

```diff
@@ -56,7 +64,8 @@ test('validate-npm-package-name', function (t) {
   t.deepEqual(validate(' leading-space'), {
     validForNewPackages: false,
     validForOldPackages: false,
-    errors: ['name cannot contain leading or trailing spaces', 'name can only contain URL-friendly characters']})
+    errors: ['name cannot contain leading or trailing spaces', 'name can only contain URL-friendly characters'],
+    warnings: ['name cannot start with a special/capital character']})
```

Comment:
This test case is testing for start with whitespace, so this messaging seems inaccurate.

Reply:
So, what happens is each package name gets checked against all the rules created in the index.js, and errors and warnings got pushed in the list so while writing a test case, we have to mention all the warnings. That' why there are 2 warnings here.